drivers/ovsdriver.go

```diff
@@ -40,11 +40,19 @@ const (
 	hostPortName = "contivh0"
 )
 
+//EpInfo contains the ovsport and id of the group
+type EpInfo struct {
+	Ovsportname string
+	EpgKey      string
+	PacketTag   string
```

Comment:
Please rename this field PacketTagType to be consistent with other places.. PacketTag refers to vlan-id..